### ISSUE TITLE
Predefined m/z tables for MS1 annotation

### ISSUE THREAD

**Issue by michaelwitting** [Metabolomics-MPC/NTAnnotationWorkflow#30]
In some cases users might have already precalculated m/z tables, e.g. for substance ionizing with adducts not covered at the moment in `MetaboCoreUtils` or only measurable as in-source fragment.
We should give the option for users to use also precalculated tables. What do you think @chufz ?
I have already an idea how to implement this.

**Comment by chufz** [comment]
Yes, maybe have a second table also with specific RT and ppm windows for each suspect in that table?

**Comment by michaelwitting** [comment]
That might be a bit to complicated, since we have to construct the `param` object again and again. Dependent on the number of compounds that might influence the performance.

**Comment by michaelwitting** [comment]
My suggestion is to check if the read data contains mz instead of exact_mass. If yes, we would use the `MzParam` or `MzRtParam`, if no everything stays the same as now.
I will try to implement this.
